controllers/profile.js

```diff
@@ -103,4 +95,31 @@ export default class ProfileController {
       });
     }
   }
+
+  /**
+   * @description - updates user's profile
+   * @static
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @memberof ProfileController
+   * @returns {object} User Profile
+   */
+  static async updateProfileImage(req, res) {
+    const { user: { id }, body: { images } } = req;
+    const image = images[0];
+    try {
+      const [, [{
+        name, email, username, bio, image: profilePic, createdAt, updatedAt
+      }]] = await User.update({ image }, { returning: true, raw: true, where: { id } });
+      return res.status(205).json({
```

Comment:
Great job Michael. Is there a reason why you're returning 205? I am of the opinion that a 201 or 200 would be appropriate here. According to https://httpstatuses.com/205, a 205 response code would imply that no additional content will be provided, and you're providing a body in your response. Kindly look into this.

Reply:
Thank you for this feedback. I have made use of the material you shared and I now have a better understanding of the appropriate error response to use for this request.